### DISCUSSION TITLE
Start API

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5862] [Q&A]
For example i have create clients module, i want to create API and use it, please how to start using API in statamic? JUST brief to understand it

**Comment by edalzell** [comment]
Statamic is a Laravel package, check out the Laravel docs for things like that, unless you're talking about Statamic's Content API?

**Reply by HazemMohamed252** [reply]
I don not mean Statamic's Content API, i mean pure APIs from scratch
